client/commands.c

```diff
@@ -494,20 +494,18 @@ void processNetCommand(char *nextParam, FILE *out)
 	xdag_net_command(netcmd, out);
 }
 
-void processRpcWhiteListCommand(char *nextParam, FILE *out)
+void processRPCCommand(char *nextParam, FILE *out)
 {
-	char *method = strtok_r(nextParam, " \t\r\n", &nextParam);
-	if(!method) {
-		fprintf(out, "white: method is not given.\n");
-		return;
-	}
-	char *address = strtok_r(0, " \t\r\n", &nextParam);
-	if(!address && 0 != strcmp(method, "-l")) {
-		fprintf(out, "white: address not given.\n");
-		return;
+	char *cmd = NULL;
+	char rpccmd[4096] = {0};
+	*rpccmd = 0;
```

Comment:
I thought `char rpccmd[4096] = {0};` fills the array with zeroes. If `*rpccmd = 0;` necessary?